keps/sig-release/k8s-image-promoter.md

```diff
@@ -4,7 +4,7 @@ authors:
   - "@javier-b-perez"
 owning-sig: sig-release
 participating-sigs:
-  - TBD
+  - wg-k8s-infra
```

Comment:
This feels odd, given that wg-k8s-infra is not a SIG.
Can we maybe go for `participating` or `participating-groups` instead?

Reply:
Don't want to change the key here in this PR, we should do a follow up to rename